cmd/hubd/main.go

```diff
@@ -92,6 +92,14 @@ var (
 				Key:      "addr.powergate.api",
 				DefValue: "",
 			},
+			"addrAnalyticsApi": {
+				Key:      "addr.analytics.api",
+				DefValue: "",
+			},
+			"addrFilrewardsApi": {
```

Comment:
@textileben I missed this one before. Hub connects to the fil rewards service so here is the new address config for that.